customer-wallet/src/containers/Reviews/index.js

```diff
@@ -0,0 +1,163 @@
+          name: review.name,
+          address: review.address,
+          telephone: null,
+          categories: review.types,
+          location: {
+            lat: review.latitude,
+            lng: review.longitude
+          },
+          url: review.webpage,
+        },
+        platform: {
+          name: 'yelp',
+          url: 'yelp.com',
+          subject_url: review.url
+        },
+        author: {
+          name: review.author,
+        },
+        review: {
+          date_published: review.datePublished,
+          url: review.rev_url,
+          title: null,
+          text: review.description
+        },
+        rating: {
+          min: 1,
+          value: review.ratingValue,
+          max: 5,
+        },
+        verifiable: false,
+        proof: null
+      }
+    })
+  }
+
+  transformTripAdvisorData() {
+    return map(tripAdvisorData, (review) => {
+      return {
+        subject: {
+          name: review.name,
+          address: `${review.street}, ${review.city} ${review.postal}, ${review.country}`,
+          telephone: review.telephone,
+          categories: [review.type, review.subtype],
+          location: null,
+          url: review.url,
+          platform_url: review.restaurant_url,                    
+        },
+        platform: {
+          name: 'TripAdvisor',
+          url: 'tripadvisor.com',
+          subject_url: review.url
+        },
+        author: {
+          name: review.author,
+        },
+        review: {
+          date_published: review.date,
+          url: review.rev_url,
+          title: review.title,
+          text: review.text
+        },
+        rating: {
+          min: 1,
+          value: review.score,
+          max: 5,
+        }
+      }
+    })
+  }
+
+  transformUpworkData() {
+    return map(upworkData, (review) => {
+      let detailed_review = map(review.feedback.scoreDetails, (detail) => {
+        return {
+          rating: {
+            min: 0,
+            value: detail.score,
+            max: 5,
+          },
+          review: detail.description,
+          category: detail.label
+        }
+      })
+      return {
+        subject: {
+          name: review.agencyName,
+          categories: review.skills,
+        },
+        platform: {
+          name: 'Upwork',
+          url: 'upwork.com',
+        },
+        author: {
+          name: review.clientId,
+        },
+        review: {
+          text: review.feedback.comment
+        },
+        rating: {
+          min: 1,
+          value: review.feedback.score,
+          max: 5,
+        },
+        detailed_review
+      }
+    })
+  }
+
+  renderReviews(reviews, classes) {
+    return <Grid container spacing={16} className={classes.root}>
+            {reviews.map((review, index) => (
+               <Grid key={index} item xs={12} lg={6} > 
+                 <Review review={review} index={index + 1} />
+               </Grid>
+             ))}
+    </Grid>
+  }
+  
+  render() {
```

Comment:
I imagine that we will eventually move the transform code somewhere else so that the react frontend already received normalized reviews